homework/lesson2/LZDCA-HmWk-2-2.py

```diff
@@ -0,0 +1,23 @@
+# HOMEWORK: LESSON 2 : VARIABLES
+# THE CUSTOMER INPUTS ARE SKIPED. THEY ARE ASSIGNED HERE
+
+import statistics
```

Comment:
cool, although we haven't gone through libraries yet

Reply:
I looked for help online. I like python, a lot of people are using it and a lot of help I can get

Reply:
yea, looking for help online is one of the most frequent thing software developers do